cli/operator/node.go

```diff
@@ -475,7 +532,63 @@ var StartNodeCmd = &cobra.Command{
 	},
 }
 
-func validateConfig(nodeStorage operatorstorage.Storage, networkName string, usingLocalEvents bool) error {
+func privateKeyFromKeystore(privKeyFile, passwordFile string) (keys.OperatorPrivateKey, []byte, error) {
+	// nolint: gosec
+	encryptedJSON, err := os.ReadFile(privKeyFile)
+	if err != nil {
+		return nil, nil, fmt.Errorf("could not read PEM file: %w", err)
+	}
+
+	// nolint: gosec
+	keyStorePassword, err := os.ReadFile(passwordFile)
+	if err != nil {
+		return nil, nil, fmt.Errorf("could not read password file: %w", err)
+	}
+
+	decryptedKeystore, err := keystore.DecryptKeystore(encryptedJSON, string(keyStorePassword))
+	if err != nil {
+		return nil, nil, fmt.Errorf("could not decrypt operator private key keystore: %w", err)
+	}
+
+	operatorPrivKey, err := keys.PrivateKeyFromBytes(decryptedKeystore)
+	if err != nil {
+		return nil, nil, fmt.Errorf("could not extract operator private key from file: %w", err)
+	}
+
+	return operatorPrivKey, decryptedKeystore, nil
+}
+
+func assertSigningConfig(logger *zap.Logger) (usingSSVSigner, usingKeystore, usingPrivKey bool) {
+	providedCount := 0
+	if cfg.SSVSignerEndpoint != "" {
+		providedCount++
+		usingSSVSigner = true
+	}
+	if cfg.KeyStore.PrivateKeyFile != "" || cfg.KeyStore.PasswordFile != "" {
+		if cfg.KeyStore.PrivateKeyFile == "" || cfg.KeyStore.PasswordFile == "" {
+			logger.Fatal("both keystore and password files must be provided if using keystore")
+		}
+		providedCount++
+		usingKeystore = true
+	}
+	if cfg.OperatorPrivateKey != "" {
+		providedCount++
+		usingPrivKey = true
+	}
+
+	if providedCount != 1 {
+		logger.Fatal("expected only one signing method to be configured",
+			zap.String("ssv_signer_endpoint", cfg.SSVSignerEndpoint),
+			zap.String("private_key_file", cfg.KeyStore.PrivateKeyFile),
+			zap.String("password_file", cfg.KeyStore.PasswordFile),
+			zap.Bool("has_operator_private_key", cfg.OperatorPrivateKey != ""),
+		)
+	}
+
+	return usingSSVSigner, usingKeystore, usingPrivKey
+}
```

Comment:
can we make the error messages more user friendly?

for example:
- "cannot enable both remote signing (SSVSignerEndpoint) and local signing (PrivateKeyFile/OperatorPrivateKey)"
- "cannot enable both OperatorPrivateKey and PrivateKeyFile"

Reply:
sure, done